tests/cli.test.ts

```diff
@@ -40,16 +40,16 @@ describe('CLI', function() {
 
     expect(result)
       .to.have.string('Source\tSize')
-      .and.have.string('dist/bar.js\t2854');
+      .and.have.string('dist/bar.js\t104');
   });
 
   it('should output multiple results as tsv', async function() {
     const result = await execute(SCRIPT_PATH, ['data/inline-map.js', 'data/foo.min.js*', '--tsv']);
 
     expect(result)
       .to.have.string('Source\tSize')
-      .and.have.string('dist/bar.js\t2854')
-      .and.have.string('dist/bar.js\t97');
+      .and.have.string('<unmapped>\t2309')
```

Comment:
The size of `dist/bar.js` is now the same in both files, while the difference in in the unmapped size.